.github/workflows/codeql_analysis.yml

```diff
@@ -1,30 +1,30 @@
-# For most projects, this workflow file will not need changing; you simply need
-# to commit it to your repository.
-#
-# You may wish to alter this file to override the set of languages analyzed,
-# or to provide custom queries or build logic.
 name: 'CodeQL'
 
 on:
   push:
-    branches:
-      - master
-      - next
+    branches: [ "master", "next", "master-bm" ]
   pull_request:
```

Comment:
I think we can change this to:

```
  pull_request:
    types: [ opened, synchronize ]
```
I think they do exactly the same thing but I find it more readable.

Reply:
OK, I will amend this as that makes sense